### DISCUSSION TITLE
降智issues“整合包”

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3200] [讨论]
**这不是放寒假了嘛，比较多的电脑小白、MC小白会经常使用启动器，触发一些“Bug”，想到一些“灵感”，特地跑过来发issue……**

要么就是自己电脑的问题，游戏崩了反馈启动器的问题，下载源崩了也反馈启动器的问题，反正~~五彩斑斓~~（？
要么就是确实有这个Bug，但是已经反馈过了，反馈者又不去搜有没有重复内容的issue、看常见问题列表，**甚至那个问题都Pin了**
要么就是从不管这个Bug是否有着合理的触发方式，就是要反馈issue
要么就是《异响天开》的新功能提议或优化，从不管这是否有用，坚持要加
要么就是……

算了，先写这么多，之后再补充？

**Comment by WorldHim** [comment]
所以这个讨论的意义在何？

**Reply by allMagicNB** [reply]
遇到过的离谱issue（

**Reply by WorldHim** [reply]
> 遇到过的离谱issue（

那这种意义不大的讨论就别开了，毕竟这不是论坛。。

**Reply by allMagicNB** [reply]
> > 遇到过的离谱issue（
> 
> 那这种意义不大的讨论就别开了，毕竟这不是论坛。。

没事
~~反正挂着没啥影响~~（？

**Comment by WForst-Breeze** [comment]
个人感觉没必要，谁都有是萌新的时候，都是可以理解的

**Reply by allMagicNB** [reply]
> 个人感觉没必要，谁都有是萌新的时候，都是可以理解的

没有特意嘲笑的意思，只是单纯的收集一下……

**Reply by ghost** [reply]
haha

**Comment by Light-Beacon** [comment]
> > 个人感觉没必要，谁都有是萌新的时候，都是可以理解的
> 
> 没有特意嘲笑的意思，只是单纯的收集一下……

感觉这种收集就比较负面，显得不是很新手友好，还是别在这公开收集的好（

**Reply by allMagicNB** [reply]
啊这……
还没收集呢（不是
话说确实有点不尊重人了（

**Comment by ghost** [comment]


**Reply by allMagicNB** [reply]
龙猫做啥了？
